components/ui/Anchor.jsx

```diff
@@ -5,7 +5,7 @@ export default function Anchor({ children, url }) {
         <Link
             href={url}
             target="_blank"
-            className="inline-block font-medium sm:text-sm text-xs -tracking-tight leading-3.75 sm:leading-4.25 text-black underline underline-offset-3 hover:text-secondary"
+            className="inline-block font-medium sm:text-sm text-xs -tracking-tight sm:leading-4.25 leading-4.24 text-black underline underline-offset-3 hover:text-secondary"
```

Comment:
Add the class of leading-4.24 in the tailwind config file, or check if the class was misspelled.